mock-relying-party-ui/README.md

```diff
@@ -4,7 +4,6 @@ This react application is the UI for the mock relying party portal
 
 ## Overview
 
-This repository contains the reference implementation of a relying party's website that wants to use [MOSIP's
 This repository contains the reference implementation of a relying party's website that wants to use [MOSIP's Esignet (Identity Provider) services](https://github.com/mosip/esignet) to log in users into its portal.
```

Comment:
i think it will better if we change this sentence to say that mock-relying-party-ui can be used with any OIDC protocol compliant Identity providers